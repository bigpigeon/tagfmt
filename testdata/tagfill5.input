//gofmt -f "json=lit_camel(:field)"

package main

type OrderDetail struct {
    ID        string   ``
    User_Name string   ``
    OrderID   string   ``
    Callback  string   ``
    Address   []string ``
}
