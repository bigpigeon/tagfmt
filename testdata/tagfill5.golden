//gofmt -f "json=lit_camel(:field)"

package main

type OrderDetail struct {
	ID        string   `json:"iD"`
	User_Name string   `json:"userName"`
	OrderID   string   `json:"orderID"`
	Callback  string   `json:"callback"`
	Address   []string `json:"address"`
}
